boot/cypress/MCUBootApp/main.c

```diff
@@ -121,6 +126,12 @@ int main(void)
         if (fih_eq(fih_rc, FIH_SUCCESS))
         {
             BOOT_LOG_INF("User Application validated successfully");
+            /* intitalize watchdog timer. it should be updated from user app
```

Comment:
Also, stylistically, the rest of the code in MCUboot usually has '*' on each line.